subsys/bluetooth/services/fast_pair/use_case/Kconfig

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menu "Fast Pair use case configuration"
```

Comment:
Do we need menu for only one choice?

Reply:
I was expecting that this option could grow with time while we add helper modules with code to the use case layer

Reply:
We can always reorganize Kconfig options layout a bit when it's needed.